src/components/Login.tsx

```diff
@@ -0,0 +1,96 @@
+import { Button, Flex, Input, Text, Link, Image } from '@chakra-ui/react'
+
+const Login = () => {
+  return (
+    <Flex
+      width="100%"
+      maxWidth="390px"
```

Comment:
Como voce ta colocando um `maxWidth` seria bom colocar um `margin: 0 auto`

Reply:
Se for assim, troca para margin-inline, que ai vai alinhar apenas na vertical